apps/editor/src/app/locales/de.json

```diff
@@ -715,6 +715,7 @@
       "authors": "Autor*innen",
       "blocks": {
         "polls": "Umfrage",
+        "blockStyles": "Block Styling",
```

Comment:
i think some translations are missing here

Reply:
also, no french translations

Reply:
Thats intended, it's like the translation I copied. We should define a workflow for this as it used to be that we just do english and Laurent does the rest.

We have a few that we only have in english.